contracts/crowdloan/Crowdloan.sol

```diff
@@ -132,20 +129,6 @@ contract Crowdloan is Initializable, ICrowdloan, ReentrancyGuard {
         emit Fund(msg.sender, amount);
     }
 
-    /// @notice Get a refund for a debt token owned by the sender
```

Comment:
@tspoff  I thought the `refund` will be put behind `cancelled`, to allow contributors withdraw if Ines decides to reject the loan.
Is this going to be handled manually?